programs/server/play.html

```diff
@@ -532,6 +533,18 @@
         document.getElementById('user').value = user_from_url;
     }
 
+    const pass_from_url = current_url.searchParams.get('password');
+    if (pass_from_url) {
+        document.getElementById('password').value = pass_from_url;
+        if (!opened_locally) {
```

Comment:
Why shouldn't we apply it if opened_locally?

Reply:
Browsers don't allow manipulating history for a file:// protocol.